web_ui/frontend/app/origin/page.tsx

```diff
@@ -57,7 +57,8 @@ export default function Home() {
                                         justifyContent:"center",
                                         display:"flex",
                                         bgcolor:"white",
-                                        borderRadius:2
+                                        borderRadius:2,
+                                        p: 1
```

Comment:
Do we want to do the same padding for the rest of pages? i.e. registry and director?

Reply:
@haoming29 This page was the outlier, the others all ready have padding.